src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/tests/poc/validator/PocValidator.java

```diff
@@ -0,0 +1,63 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.integTests.tests.poc.validator;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.integTests.framework.api.IValidator;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import java.util.List;
+import org.jooq.Record;
+import org.jooq.Result;
+import org.junit.Assert;
+
+// Validators are only initialized once to evaluate a test method.
+public class PocValidator implements IValidator {
+  long startTime;
+
+  public PocValidator() {
+    startTime = System.currentTimeMillis();
+  }
+
+  /**
+   * {"rca_name":"ClusterRca",
+   * "timestamp":1596557050522,
+   * "state":"unhealthy",
+   * "HotClusterSummary":[
+   * {"number_of_nodes":1,"number_of_unhealthy_nodes":1}
+   * ]}
+   */
+  @Override
+  public boolean check(JsonElement response) {
+    JsonArray array = response.getAsJsonObject().get("data").getAsJsonArray();
+    if (array.size() == 0) {
+      return false;
+    }
+
+    for (int i = 0; i < array.size(); i++) {
+      JsonObject object = array.get(i).getAsJsonObject();
+      if (object.get("rca_name").getAsString().equals("ClusterRca")) {
+        return checkClusterRca(object);
+      }
+    }
+    return false;
+  }
+
+  /**
+   * "HotClusterSummary":[{"number_of_nodes":1,"number_of_unhealthy_nodes":1}]
```

Comment:
Either this comment is incorrect or the call to object.get("state") down below is incorrect

Reply:
The comment doesn't capture the full object but just one of the k/v pairs. I have updated the comment to include the full object. This should make it more clear.